src/Ilios/CoreBundle/DataFixtures/ORM/LoadCompetencyData.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Ilios\CoreBundle\DataFixtures\ORM;
+
+use Doctrine\Common\DataFixtures\DependentFixtureInterface;
+
+use Ilios\CoreBundle\Entity\Competency;
+
+/**
+ * Class LoadCompetencyData
+ * @package Ilios\CoreBundle\DataFixtures\ORM
+ */
+class LoadCompetencyData extends AbstractFixture implements DependentFixtureInterface
+{
+    public function __construct()
+    {
+        parent::__construct('competency');
+    }
+    /**
+     * {@inheritdoc}
+     */
+    protected function createEntity(array $data)
+    {
+        // `competency_id`,`title`,`parent_competency_id`,`school_id`
+        $entity = new Competency();
+        $entity->setId($data[0]);
+        $entity->setTitle($data[1]);
+        if (! empty($data[2])) {
+            $entity->setParent($this->getReference($this->getKey() . $data[2]));
```

Comment:
In this case we would need to guarantee that the parent loaded before the child.  It might be worth checking for the existence of this reference and throwing a nice exception because I fear that if we change the CSV it would be hard to figure out what was broken.